### PR TITLE
DCS-411: CA can view the DMs HDC decision comments

### PR DESCRIPTION
**Pull request by GurnankCheema**
The comments appear only after the DM had made a decision providing the decision is a refusal.
The comments appear in the orange bordered box at the top of the CA tasklist.

### CODE REVIEW

File: server/utils/licenceStatus.js [hunk 1]
```diff
@@ -176,6 +179,7 @@ function getCaStageState(licence) {
       bassAccepted,
       bassWithdrawn,
       bassWithdrawalReason,
+      caRefused: (approvalRelease.decision === 'Yes' && finalChecksRefusal.decision === 'Yes') || false,
```
Comment (bell-pa) [inline server/utils/licenceStatus.js:182]: You don't need the `|| false` bit. The value of the  first part of the expression is always either `true` or `false`

File: server/views/taskList/taskListTemplate.pug [hunk 2]
```diff
@@ -23,9 +23,20 @@ block content
   else if licenceStatus.decisions.unsuitableResult
     div.error-banner.marginTopMedium
       strong.bold The offender is presumed unsuitable for HDC release
-  else if user.role != 'DM' && licenceStatus.decisions.refused
+  else if user.role === 'CA' && licenceStatus.decisions.caRefused
     div.error-banner.marginTopMedium
-      strong.bold Home detention curfew refused - #{licenceStatus.decisions.refusalReason}
+      strong.bold Home detention curfew refused by Case Admin - #{licenceStatus.decisions.refusalReason}
+  else if user.role === 'CA' && licenceStatus.decisions.refused && licenceStatus.decisions.decisionComments
```
Comment (bell-pa) [inline server/views/taskList/taskListTemplate.pug:29]: Prefer the full name of the role to an abbreviated form: 'Home detention curfew refused by Prison Case Administrator'

File: server/utils/licenceStatus.js (hunk 1 above)
Comment (GurnankCheema) [reply server/utils/licenceStatus.js:182]: oops, that was me trying code out but then forgot to take it out.
Done

File: server/views/taskList/taskListTemplate.pug (hunk 2 above)
Comment (GurnankCheema) [reply server/views/taskList/taskListTemplate.pug:29]: fair point
Done

Comment (GurnankCheema) [reply server/views/taskList/taskListTemplate.pug:29]: changed to 'prison case admin' and 'decision maker'

**Pull request merged**